test/unit_tests/components/list/conversationListCallingCellSpec.js

```diff
@@ -27,9 +27,9 @@ import 'src/script/components/list/conversationListCallingCell';
 import {User} from 'src/script/entity/User';
 
 function createCall(state) {
-  const call = new Call();
+  const selfParticipant = new Participant('selfId');
+  const call = new Call('', '', undefined, selfParticipant);
```

Comment:
Let's use `createRandomUuid` to pass IDs to the call so it looks a little more like on production.